lib/common/schemas.c

```diff
@@ -177,13 +177,13 @@ schema_filter(const struct dirent *a)
     pcmk__schema_version_t version = SCHEMA_ZERO;
 
     if (!g_str_has_prefix(a->d_name, "pacemaker-")) {
-        /* crm_trace("%s - wrong prefix", a->d_name); */
+        // pcmk__trace("%s - wrong prefix", a->d_name);
 
     } else if (!g_str_has_suffix(a->d_name, ".rng")) {
-        /* crm_trace("%s - wrong suffix", a->d_name); */
+        // pcmk__trace("%s - wrong suffix", a->d_name);
 
     } else if (!version_from_filename(a->d_name, &version)) {
-        /* crm_trace("%s - wrong format", a->d_name); */
+        // pcmk__trace("%s - wrong format", a->d_name);
```

Comment:
These three have all been commented out since their introduction in 2016 (ae81cdf070d).  I think we can get rid of them too.